shell/app/common/components/contractive-filter.scss

```diff
@@ -102,6 +102,11 @@
     background-color: transparent;
   }
 
+  .ant-dropdown-menu-item,
+  .ant-dropdown-menu-submenu-title {
+    color: $color-dark-65;
+  }
+
```

Comment:
I have one question. Why not set the menu option className?

Reply:
The previous code has added a ClassName to the Dropdown overlayClassName, so I used that className.

Reply:
I mean why not change className of `Menu.Item`
![image](https://user-images.githubusercontent.com/5175455/125757941-bb40c328-95b8-4a9f-bb85-1c2d4abc7ddf.png)